Fix header conversion in eachBatch (#130)
confluentinc/confluent-kafka-javascript: CHANGELOG.md

> # confluent-kafka-javascript v0.4.0

Reviewer: ```suggestion
# confluent-kafka-javascript v0.3.1
```

> v0.4.0 is a limited availability feature release. It is supported for all usage.

Reviewer: ```suggestion
v0.3.1 is a limited availability maintenance release. It is supported for all usage.
```

> ## Bug Fixes

Reviewer: This is the summary for fixes and new features, let's put enhancements like in .NET client
```suggestion
## Enhancements
```
